stdlib/enum.pyi

```diff
@@ -193,14 +193,11 @@ class IntEnum(int, _IntEnumBase):
 
 def unique(enumeration: _EnumerationT) -> _EnumerationT: ...
 
-_auto_null: Any
-
-# subclassing IntFlag so it picks up all implemented base functions, best modeling behavior of enum.auto()
+# This is a lie, auto() does not derive from IntFlag. But auto instances
+# are solely intended to be used inside the body of Enum sub-classes
+# and get replaced with Enum instances at runtime.
 class auto(IntFlag):
-    _value_: Any
-    @_magic_enum_attr
-    def value(self) -> Any: ...
```

Comment:
Why delete these?

Reply:
`IntFlag` has them as `int`, though, which isn't correct for `auto`. The `_value_` and `value` attributes of enum members created using `auto()` depends on the return type of `_generate_next_value_`, which can be overriden in custom enum classes.